.travis.yml

```diff
@@ -13,12 +13,17 @@ addons:
 
 install:
   - npm install
-  - npm install -g bower
-  - bower install
+  - polymer install --variants
 
 before_script:
   - gulp lint
-  - ([ "$TRAVIS_EVENT_TYPE" = "pull_request" ] || TRAVIS_BRANCH=quick/${TRAVIS_BUILD_ID} xvfb-run -s '-screen 0 1024x768x24' wct)
+  - ([ "$TRAVIS_EVENT_TYPE" = "pull_request" ] || TRAVIS_BRANCH=quick/${TRAVIS_BUILD_ID} xvfb-run -s '-screen 0 1024x768x24' polymer test)
 
 script:
+  # test against Polymer 1
+  - xvfb-run -s '-screen 0 1024x768x24' wct
```

Comment:
Use `polymer test` here and remove the moving of `bower_components`

`polymer test` will test both variants.